doc/api/process.markdown

```diff
@@ -803,6 +803,35 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.resourceUsage();
+
+Returns the resource usage measures
+
+Note: Not all resource usage measures are available on Windows. Fields
+that aren't available are zero.
+
+An example of this object looks like:
```

Comment:
Maybe just say "example output looks like" since no mention of an object was made prior to this.